### PR TITLE
feat/add-house-contract

### PR DESCRIPTION
**Pull request by Keith-CY**

### CODE REVIEW

File: contracts/House.sol [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    event ProposalDenied(uint indexed idx, address indexed director);
+
+    EconomicalModel economicalModel;
+
+    address public executor;
+
+    modifier memberRequired() {
+        uint i = 0;
+        for(i = 0; i < senators.length; i++) {
+            if (senators[i].addr == msg.sender) {
```
Comment (kaikai1024) [inline contracts/House.sol:74]: there is no meaning use return after `_`. What do you want?

**Review by kaikai1024 — CHANGES_REQUESTED** [review]

File: contracts/House.sol (hunk 1 above)
Comment (kaikai1024) [reply contracts/House.sol:74]: check this:

```
Explicit returns from a modifier or function body only leave the current modifier or function body. Return variables are assigned and control flow continues after the “_” in the preceding modifier.
```

Comment (Keith-CY) [reply contracts/House.sol:74]: @kaikai1024 Yep I want it to leave the loop in the modifier.

Comment (kaikai1024) [reply contracts/House.sol]: ok

Comment (kaikai1024) [reply contracts/House.sol]: ok

**Review by kaikai1024 — APPROVED** [review]

**Pull request merged**